auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -343,7 +343,8 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
     if (this.isTemporaryBusiness(business)) {
       return this.tempDescription(business)
     }
-    return GetCorpFullDescription(business.corpType.code as CorpTypeCd)
+    const code: unknown = business.corpType.code
+    return GetCorpFullDescription(code as CorpTypeCd)
```

Comment:
Can't directly cast one type to another without doing this.

Reply:
How was this working before? Or are you fixing something that wasn't working before?

Reply:
`business.corpType.code` was previously a string, but now it's of type CorpTypes.